CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- [breaking] Support Storybook 9
+- [breaking] Output ESM instead of CJS (needed to consume Storybook 9 types)
```

Comment:
Maybe the CJS -> ESM switch isn't really breaking, as it should be transparent to Node 18+ users. But it is a major change, so seemed right to label it this way.